.buildkite/cache-builder.yml

```diff
@@ -0,0 +1,23 @@
+# This pipeline runs via Buildkite's scheduled jobs feature.
```

Comment:
💄  So that you get syntax highlighting and autocomplete in IDEs like Visual Studio:
```suggestion
# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
---
# This pipeline runs via Buildkite's scheduled jobs feature.
```